src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.js

```diff
@@ -95,6 +93,13 @@ app.directive('discoverField', function ($compile) {
         }
       };
 
+      $scope.onKeyUp = function onKeyUp(e, field) {
+        // Support keyboard accessibility by toggling display on ENTER or SPACE keypress.
+        if ([13, 32].includes(e.keyCode)) {
+          $scope.toggleDetails(field);
+        }
+      };
```

Comment:
`src/ui/public/directives`?

I suggested `kbn-buttonPress` because it is kind of similar to the angular event handling directives (`ng-click`, `ng-keyup` etc.) which we would be mimicking and a "buttonPress" (pressing space or enter on a button) is what we're trying to emulate.